deploy.sh

```diff
@@ -15,5 +15,5 @@ ansible-galaxy install -r requirements.yml
 ansible-playbook environments.yml
 ansible-playbook --limit=epoch setup.yml
 # split monitoring playbook by environment because Ansible group vars are merged otherwise
-ansible-playbook --limit='uat:&epoch:!tester' --extra-vars "datadog_api_key=${datadog_api_key:?}" monitoring.yml
```

Comment:
Spurious change?

Reply:
I mentioned in the PR description that the tester node is already monitored, this just reflects the current state.